src/Pelagos/Bundle/AppBundle/Resources/public/js/external-download-log.js

```diff
@@ -0,0 +1,31 @@
+var $ = jQuery.noConflict();
+
+//FOUC preventor
+$("html").hide();
+
+$(document).ready(function () {
+    "use strict";
+
+    $("html").show();
+
+    $(".pelagosNoty").pelagosNoty({timeout: 0, showOnTop:false});
+
+    let userType = $("#externalDownloadLog_userType");
+
+    checkUserType(userType);
+
+    userType.on("change", function () {
+        if (userType.val() === "0") {
```

Comment:
A user should be able to toggle back and forth and have the username field re-appear if they go from "no" back to "yes". (user might accidentally toggle it and have no way to get it back)

Reply:
Added else case